jwst/assign_wcs/nirspec.py

```diff
@@ -1631,7 +1631,7 @@ def _is_valid_slit(domain):
         ylow, yhigh = domain[1]
         if (xlow >= 2048 or ylow >= 2048 or
             xhigh <= 0 or yhigh <= 0 or
-                xhigh - xlow < 1 or yhigh - ylow < 1):
+                xhigh - xlow < 2 or yhigh - ylow < 1):
```

Comment:
It makes sense in this particular context to change the requirements for the `xlow` to `xhigh` range, because for NIRSpec data that's the wavelength direction. But do we need to allow for cases where the dispersion is along the y-axis and hence the check should be flipped to look at the `ylow` to `yhigh` range? MIRI LRS, for example, have dispersion along the y axis.

Reply:
This function is only called in the NIRSpec FS and MSA WCS computation.  MIRI LRS uses a separate module.